app/validators/wpcc/salary_threshold_validator.rb

```diff
@@ -0,0 +1,22 @@
+module Wpcc
+  class SalaryThresholdValidator < ActiveModel::Validator
+    SALARY_THRESHOLDS  = [5876.00, 489.67, 452.00, 113.00].freeze
+    SALARY_FREQUENCIES = %w[year month fourweeks week].freeze
+    LOW_SALARY_THRESHOLDS = SALARY_FREQUENCIES.zip(SALARY_THRESHOLDS).to_h
```

Comment:
@willhall88 this line has been copied from the [YourDetailsForm file](https://github.com/moneyadviceservice/wpcc/blob/master/app/models/wpcc/your_details_form.rb). 

I have another commit to remove this re-assignment `SALARY_FREQUENCIES = %w[year month fourweeks week].freeze` and use `Wpcc::YourDetailsForm::SALARY_FREQUENCIES` so that it's defined in one place rather than two. 

Hard-coding the hash for `LOW_SALARY_THRESHOLDS` will require maintaining the salary_frequency list in two places.